bin/convert-argv.js

```diff
@@ -80,15 +69,15 @@ module.exports = function(...args) {
 		const configArgList = Array.isArray(argv.config) ? argv.config : [argv.config];
 		configFiles = configArgList.map(mapConfigArg);
 	} else {
-		for (i = 0; i < defaultConfigFiles.length; i++) {
-			const webpackConfig = defaultConfigFiles[i].path;
-			if (fs.existsSync(webpackConfig)) {
-				configFiles.push({
-					path: webpackConfig,
-					ext: defaultConfigFiles[i].ext
-				});
-				break;
-			}
+		const webpackConfigFileRegExp = `(webpack.config|webpackfile)(${extensions.join("|")})`;
```

Comment:
`configFileRegExp` is sufficient

Reply:
add a constant, so we can add more later, `const defaultConfigFiles = ["webpack.config", "webpackfile"]` and join them in the regexp instead like the last node in the regexp